src/text.lisp

```diff
@@ -1,28 +1,45 @@
 (in-package :trivial-clipboard)
 
+(defun executable-find (command)
+  "Search for COMMAND in the PATH and return the absolute file name.
+Return nil if COMMAND is not found anywhere."
+  (multiple-value-bind (path)
+      (ignore-errors
+       (uiop:run-program (format nil "command -v ~A" command)
+                         :output '(:string :stripped t)))
+    path))
+
 (defvar *clipboard-in-command*
   #+(or darwin macosx)
   "pbcopy"
   #+(and :unix (:not :darwin))
-  "xclip")
+  (or (executable-find "xclip")
+      (executable-find "xsel")))
 
 (defvar *clipboard-out-command*
   #+(or darwin macosx)
   "pbpaste"
   #+(and :unix (:not :darwin))
-  "xclip")
+  (or (executable-find "xclip")
+      (executable-find "xsel")))
 
```

Comment:
if ``xclip`` isn't installed ``executable-find`` will be called fourtimes. can it be ``*clipboard-in-command*`` instead?